ee/tabby-webserver/src/service/mod.rs

```diff
@@ -311,9 +311,11 @@ pub async fn create_gitlab_client(
     // Gitlab client expects a url base like "gitlab.com" not "https://gitlab.com"
     // We still want to take a more consistent format as user input, so this
     // will help normalize it to prevent confusion
```

Comment:
```suggestion
```

Since it seems the new version of `gitlab` no longer has this quirk, we should remove this comment

Reply:
Does it mean we can pass in "https://..." or "http://..." directly, without having to set `insecure` by ourselves?

Reply:
Potentially, but I think it's still a good idea for correctness

Reply:
https://docs.rs/gitlab/latest/src/gitlab/gitlab.rs.html#417-502 Seems it will change it to https unless insecure is set

Reply:
ok - then will keep the current implementation